lib/index.js

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
I have no idea why it's there in the first place. We do have a [LICENSE](https://github.com/sinonjs/sinon/blob/master/LICENSE), so repeating it in every file seems superfluous. It should be enough with a short description of the module. The rest is redundant.